File: 프로그래머스/0/181874. A 강조하기/A 강조하기.cpp
#include <bits/stdc++.h>

using namespace std;

string solution(string myString) {
    for(char &ch : myString){
        if(ch == 'a') ch = 'A';
        else if('A' < ch && ch <= 'Z') ch = std::tolower(ch);
    }
    return myString;
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.java
import java.io.*;
import java.util.*;

class Solution {
    public String solution(String myString) {
        StringBuilder sb = new StringBuilder(myString.length());
        for(char ch : myString.toCharArray()){
            if(ch == 'a') sb.append('A'); 
            else if('A' < ch && ch <= 'Z') sb.append(Character.toLowerCase(ch));
            else sb.append(ch);
        }
        return sb.toString();
    }
}